File: ch13/Chapter13-WCFService-CS/App_Code/ICalculator.cs
using System.ServiceModel;

[ServiceContract]
public interface ICalculator
{
    [OperationContract]
    int Add(int a, int b);
          
    [OperationContract]
    int Subtract(int a, int b);
          
    [OperationContract]
    int Multiply(int a, int b);
          
    [OperationContract]
    int Divide(int a, int b);

    // Code removed for clarity
}

File: ch13/Chapter13-WCFService-VB/App_Code/ICalculator.vb
Imports System.ServiceModel

<ServiceContract()>
Public Interface ICalculator
    <OperationContract()> _
    Function Add(ByVal a As Integer, ByVal b As Integer) As Integer

    <OperationContract()> _
    Function Subtract(ByVal a As Integer, ByVal b As Integer) As Integer

    <OperationContract()> _
    Function Multiply(ByVal a As Integer, ByVal b As Integer) As Integer

    <OperationContract()> _
    Function Divide(ByVal a As Integer, ByVal b As Integer) As Integer
End Interface